code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -224,6 +224,13 @@
 	spare.domutcheck()
 	spare.Move(get_step(H.loc, pick(NORTH,SOUTH,EAST,WEST)))
 
+	if(H.GetComponent(/datum/component/nanites))
+		var/datum/component/nanites/owner_nanites = H.GetComponent(/datum/component/nanites)

```

Comment:
do this first then check `if(owner_nanites)`, otherwise you're just doubling your proccalls